### DISCUSSION TITLE
public api

### DISCUSSION THREAD

**Discussion by Manubi** [oscartbeaumont/rspc#211] [rspc Help] [answered]
Hi, 
is there currently a way to generate a REST api or a graphql endpoint with rspc?
thanks!

**Comment by oscartbeaumont** [comment, marked as answer]
Sorry for the slow response. Currently, rspc does not support this but long term I want to have an official version of something like [trpc-openapi](https://github.com/jlalmes/trpc-openapi) but right now my focus on improving rspc because it can still be really rough around the edges.
